button/button.css

```diff
@@ -0,0 +1,116 @@
+button,
+a.button {
+  background-color: var(--grey-90-a10);
+  border: 0 none;
+  border-radius: 2px;
+  color: var(--grey-90);
+  display: inline-block;
+  font-family: inherit;
+  font-size: 13px;
+  font-weight: 400;
+  height: 32px;
+  line-height: 32px;
+  min-width: 132px;
+  padding: 0 8px;
+  text-align: center;
+  text-decoration: none;
+  vertical-align: middle;
+  width: auto;
+}
+
+
+/* Primary style */
+
+button.button--primary,
+a.button.button--primary {
+  background-color: var(--blue-60);
+  color: #fff;
+  font-weight: 300;
+}
+
+button.button--primary:hover,
+a.button.button--primary:hover {
+  background-color: var(--blue-70);
+}
+
+button.button--primary:active,
+a.button.button--primary:active {
+  background-color: var(--blue-80);
+}
+
+
+/* Ghost style */
+
+button.button--ghost,
+a.button.button--ghost {
+  background-color: transparent;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 16px 16px;
+  color: var(--grey-90-a80);
+  height: 32px;
+  min-width: auto;
+  padding: 8px;
+  width: 32px;
+}
+
+
+/* Micro size */
+
+button.button--micro,
+a.button.button--micro {
+  border-radius: 2px;
+  font-size: 11px;
+  height: 24px;
+  line-height: 24px;
+  padding: 0 8px;
+  min-width: auto;
+}
+
+
+/* Puffy size */
+
+button.button--puffy,
+a.button.button--puffy {
+  border-radius: 4px;
+  font-size: 15px;
+  height: 48px;
+  line-height: 48px;
+  min-width: auto;
+  padding: 0 16px;
+}
+
+
+/* Hover, Active states */
+
+button:hover,
+a.button:hover {
+  background-color: var(--grey-90-a20);
```

Comment:
`.button--ghost:hover` should be `var(--grey-90-a10)` instead of `var(--grey-90-a20)`

Reply:
the design docs do say that, but looking at real examples, they all seem to be a20

Reply:
Can you point me to an example? Because looking at the toolbar buttons they set the hover state to `hsla(240, 5%, 5%, 0.1)` which is `var(--grey-90-a10)`.

<img width="77" alt="screen shot 2018-09-19 at 9 36 32 am" src="https://user-images.githubusercontent.com/1721875/45738019-9f5c3c00-bbef-11e8-9288-122f4ac9df94.png">

Reply:
Given that I can't even find the supposed example I was working from 😬, I'll change this to --grey-90-a10

Reply:
This should be good now!